splunk-otel-android/src/main/java/io/opentelemetry/rum/internal/RumConstants.java

```diff
@@ -24,12 +24,17 @@ public class RumConstants {
 
     public static final String OTEL_RUM_LOG_TAG = "OpenTelemetryRum";
 
+    public static final AttributeKey<String> SESSION_ID_KEY = stringKey("rum.session.id");
```

Comment:
Should we rename it to `rum.session_id`? Or perhaps rename the other attribute to `rum.session.previous_id`?

Reply:
Oh, I like `rum.session.previous_id` a lot. Thanks!